/**:
  ros__parameters:
    cyberdog_bringup_test_parameter_int0: 20

/*:
  bringup_test:
    ros__parameters:
      cyberdog_bringup_test_parameter_int1: 21

cyberdog:
  bringup_test:
    ros__parameters:
      cyberdog_bringup_test_parameter_int2: 22

/cyberdog/bringup_test:
  ros__parameters:
    cyberdog_bringup_test_parameter_int3: 23
